scripts/format_arpa_lm.py

```diff
@@ -44,6 +46,9 @@
 ngram_order = int(f.readline())
 f.close()
 
+# set the memory restriction for "sort"
+sort_mem_opt = ("--buffer-size={0} ".format(args.max_memory))
```

Comment:
I see your point. I set the empty case as default (no memory limitation) which I think is reasonable(?). I tested when --max-memory option is empty and it worked.